packages/frontend/app/routes/payments.outgoing.$outgoingPaymentId.tsx

```diff
@@ -0,0 +1,224 @@
+              <div>
+                <p className='font-medium'>Receiver</p>
+                <Link
+                  className='underline text-blue-600 hover:text-blue-800 visited:text-purple-600'
+                  to={outgoingPayment.receiver}
+                >
+                  {outgoingPayment.receiver}
+                </Link>
+              </div>
+              <div>
+                <p className='font-medium'>Received Amount</p>
+                <p className='mt-1'>
+                  {formatAmount(
+                    outgoingPayment.receiveAmount.value,
+                    outgoingPayment.receiveAmount.assetScale
+                  ) +
+                    ' ' +
+                    outgoingPayment.receiveAmount.assetCode}
+                </p>
+              </div>
+              <div>
+                <p className='font-medium'>Debit Amount</p>
+                <p className='mt-1'>
+                  {formatAmount(
+                    outgoingPayment.debitAmount.value,
+                    outgoingPayment.debitAmount.assetScale
+                  ) +
+                    ' ' +
+                    outgoingPayment.debitAmount.assetCode}
+                </p>
+              </div>
+              <div>
+                <p className='font-medium'>Sent Amount</p>
+                <p className='mt-1'>
+                  {formatAmount(
+                    outgoingPayment.sentAmount.value,
+                    outgoingPayment.sentAmount.assetScale
+                  ) +
+                    ' ' +
+                    outgoingPayment.sentAmount.assetCode}
+                </p>
+              </div>
+              <div>
+                <p className='font-medium'>Error</p>
+                {outgoingPayment.error ? (
+                  <p className='mt-1 text-red-500'>{outgoingPayment.error}</p>
+                ) : (
+                  <em>None</em>
+                )}
+              </div>
+              <div>
+                {outgoingPayment.metadata ? (
+                  <>
+                    <button
+                      className='font-medium mb-1 cursor-pointer'
+                      aria-label='toggle metadata visibility'
+                      onClick={() => setShowMetadata(!showMetadata)}
+                    >
+                      {showMetadata ? '▼' : '►'} Metadata
+                    </button>
+                    {showMetadata && outgoingPayment.metadata && (
+                      <pre
+                        className='mt-1 text-sm'
+                        dangerouslySetInnerHTML={{
+                          __html: prettify(outgoingPayment.metadata)
+                        }}
+                      />
+                    )}
+                  </>
+                ) : (
+                  <>
+                    <p className='font-medium'>Metadata</p>
+                    <p className='mt-1'>
+                      <em>None</em>
+                    </p>
+                  </>
+                )}
```

Comment:
We can avoid the whole `useState` logic by leveraging the `details` element. Additionally, I would replace the `em` element with `i`. Same visual result, but they have [different meanings](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em#i_vs._em).
```suggestion
              <details>
                <summary>Metadata</summary>
                {outgoingPayment.metadata ? (
                  <pre
                    className='mt-1 text-sm'
                    dangerouslySetInnerHTML={{
                      __html: prettify(outgoingPayment.metadata)
                    }}
                  />
                ) : (
                  <i>None</i>
                )}
              </details>
```

Reply:
This is a great suggestion, thank you.

I just changed it slightly so that `None` is shown immediately without the dropdown when there is no metadata. Currently it's hidden. Also changed the `em` to `i`.

```jsx
<div>
  {outgoingPayment.metadata ? (
    <details>
      <summary>Metadata</summary>
      <pre
        className='mt-1 text-sm'
        dangerouslySetInnerHTML={{
          __html: prettify(outgoingPayment.metadata)
        }}
      />
    </details>
  ) : (
    <div>
      <p className='font-medium'>Metadata</p>
      <p className='mt-1'>
        <i>None</i>
      </p>
    </div>
  )}
</div>
```